services/telemetry/telemetry.go

```diff
@@ -777,6 +777,9 @@ func (ts *TelemetryService) trackConfig() {
 		"bulk_indexing_batch_size": *cfg.ElasticsearchSettings.BatchSize,
 		"request_timeout_seconds":  *cfg.ElasticsearchSettings.RequestTimeoutSeconds,
 		"skip_tls_verification":    *cfg.ElasticsearchSettings.SkipTLSVerification,
+		"ca":                       *cfg.ElasticsearchSettings.CA,
+		"client_cert":              *cfg.ElasticsearchSettings.ClientCert,
+		"client_key":               *cfg.ElasticsearchSettings.ClientKey,
```

Comment:
Let's use `isDefault()` for this, i don't think otherwise it will bring much value.

Reply:
Good point. Changed it.